### PR TITLE
User fix

### PR DESCRIPTION
**Pull request by mira-miracoli**
uses a separate file and loops over it. Checks if a user can be authenticated. If not user is deleted and (re)created.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -4,3 +4,7 @@
 - include_tasks: start.yml
 
 - include_tasks: configure.yml
+
+- name: Create users and vhosts
+  include_tasks: users_vhosts.yml
+  loop: "{{ rabbitmq_users }}"
```
Comment (hexylena) [inline tasks/main.yml:10]: nice, I like it

File: tasks/users_vhosts.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  community.docker.docker_container_exec:
+    container: "{{ rabbitmq_container.name }}"
+    command: 'rabbitmqctl authenticate_user "{{ item.user }}" "{{ item.password }}"'
+  ignore_errors: true
+  register: user_ok
+
+- name: Create user if authentication failed
+  community.docker.docker_container_exec:
+    container: "{{ rabbitmq_container.name }}"
+    command: 'rabbitmqctl delete_user "{{ item.user }}"
```
Comment (hexylena) [inline tasks/users_vhosts.yml]: not sure the quoting is right here? or at least in GH interface the syntax highlighting colours look off

**Review by hexylena — APPROVED** [review]
let's try it :)

File: tasks/users_vhosts.yml (hunk 2 above)
Comment (mira-miracoli) [reply tasks/users_vhosts.yml]: oh yeah I guess that can be removed

**Comment by mira-miracoli** [comment]
I still get an error because of missing disable_stats variable. I don't see it, do you?

**Pull request merged**